module7_selenium/src/test/java/bring_It_on/pages/PastebinResultPage.java

```diff
@@ -0,0 +1,33 @@
+package bring_It_on.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PastebinResultPage {
+    public String titleString;
+    public String bashString;
+    public String codeString;
+    private WebDriver driver;
+
+    public PastebinResultPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public String resultTitle() {
```

Comment:
maybe `getResultTitle` or `getPageTitle` ?
And please add javadoc for all not obvoius public methods.